.github/workflows/pr_rats_tls.yml

```diff
@@ -0,0 +1,116 @@
+
+    - name: install rats-tls
+      run: docker exec $inclavare_test bash -c 'mv /root/inclavare-containers/${{ matrix.tag }}/rats-tls /opt;
+        mkdir -p ${{ env.RATS_TLS_BINDIR }};
+        mv /root/inclavare-containers/${{ matrix.tag }}/samples/* ${{ env.RATS_TLS_BINDIR }};
+        cp ${{ env.RATS_TLS_BINDIR }}/sgx_stub_enclave.signed.so ./'
+
+    - id: random-port-generator1
+      uses: ./.github/actions/generate-random-port
+
+    - name: Test rats tls server for mutual sgx_la
+      if: ${{ contains(matrix.sgx, 'SGX2') }}
+      run: |
+        echo random-port1 ${{ steps.random-port-generator1.outputs.random-port }}
+        docker cp /etc/sgx_default_qcnl.conf  $inclavare_test:/etc/
+        docker exec $inclavare_test bash -c '${{ env.RATS_TLS_BINDIR }}/rats-tls-server -a sgx_la -v sgx_la -t openssl -c openssl -m -p ${{ steps.random-port-generator1.outputs.random-port }}' &
+        docker exec $inclavare_test bash -c 'sleep 10 && ${{ env.RATS_TLS_BINDIR }}/rats-tls-client -l debug -a sgx_la -v sgx_la -t openssl -c openssl -m -p ${{ steps.random-port-generator1.outputs.random-port }}'
+        docker restart $inclavare_test
+
+    - id: random-port-generator2
+      uses: ./.github/actions/generate-random-port
+
+    - name: Test rats tls server for mutual sgx_ecdsa
+      if: ${{ contains(matrix.sgx, 'SGX2') }}
+      run: |
+        echo random-port2 ${{ steps.random-port-generator2.outputs.random-port }}
+        docker cp /etc/sgx_default_qcnl.conf  $inclavare_test:/etc/
+        docker exec $inclavare_test bash -c '${{ env.RATS_TLS_BINDIR }}/rats-tls-server -l debug -a sgx_ecdsa -v sgx_ecdsa -t openssl -c openssl -m -p ${{ steps.random-port-generator2.outputs.random-port }}' &
+        docker exec $inclavare_test bash -c 'sleep 10 && ${{ env.RATS_TLS_BINDIR }}/rats-tls-client -l debug -a sgx_ecdsa -v sgx_ecdsa -t openssl -c openssl -m -p ${{ steps.random-port-generator2.outputs.random-port }}'
+        docker restart $inclavare_test
+
+    - id: random-port-generator3
+      uses: ./.github/actions/generate-random-port
+
+    - name: Test rats tls server for mutual nullattester and nullverifier instances
+      run: |
+        echo random-port3 ${{ steps.random-port-generator3.outputs.random-port }}
+        docker exec $inclavare_test bash -c '${{ env.RATS_TLS_BINDIR }}/rats-tls-server -a nullattester -v nullverifier -t openssl -c openssl -m -p ${{ steps.random-port-generator3.outputs.random-port }}' &
+        docker exec $inclavare_test bash -c 'sleep 10 && ${{ env.RATS_TLS_BINDIR }}/rats-tls-client -l debug -a nullattester -v nullverifier -t openssl -c openssl -m -p ${{ steps.random-port-generator3.outputs.random-port }}'
+
+    - name: Clean up the github workspace
+      if: ${{ always() }}
+      run:
+        docker restart $inclavare_test;
+        docker exec $inclavare_test bash -c 'rm -rf /root/inclavare-containers/${{ matrix.tag }}'
+
+    - name: Kill the test container
+      run: docker stop $inclavare_test
```

Comment:
Please do NOT delete `tools/docker/Dockerfile-inclavare-dev-centos8.2` and `tools/docker/Dockerfile-inclavare-dev-ubuntu18.04` in this pr